configure

```diff
@@ -327,7 +327,7 @@ if test "$gcc" -eq 1 && ($cc $CFLAGS -c $test.c) >> configure.log 2>&1; then
         fi
       fi
       if test $buildacle -eq 1; then
-        ARCH=armv8-a+crc
+        ARCH=armv8-a
```

Comment:
Is this still needed for enabling ACLE to work? Similarly, is the eabi check directly above it still needed?

Reply:
I think you are right. It is no longer needed. Detection will now always happen later. I have left the `uname=arm` part of the eabi check but the rest removed.